sci-mathematics/octave/octave-9.2.0.ebuild

```diff
@@ -180,15 +179,15 @@ src_configure() {
 			LRELEASE="$(qt6_get_bindir)/lrelease" \
 			QHELPGENERATOR="$(qt6_get_bindir)/../libexec/qhelpgenerator"
 		myeconfargs+=(
-			"--with-opengl"
+			$(use_with gui opengl)
+			$(use_with gui x)
 			"--with-qt=6"
```

Comment:
I think what @orlitzky meant here is to put it higher up, outside of this if, so we don't have to write it twice.